securedrop/journalist_app/main.py

```diff
@@ -141,7 +140,7 @@ def reply() -> werkzeug.Response:
             return redirect(url_for("col.col", filesystem_id=g.filesystem_id))
 
         g.source.interaction_count += 1
-        filename = "{0}-{1}-reply.gpg".format(
+        filename = "{}-{}-reply.gpg".format(
```

Comment:
Huh, looks like there's a format case it misses, I guess you'd have to repeatedly apply it until you hit a steady state.

Reply:
I think this falls under "pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are sufficiently complicated (as this can decrease readability)." (from https://github.com/asottile/pyupgrade#f-strings)

You end up with a 92-char line:
```
        filename = f"{g.source.interaction_count}-{g.source.journalist_filename}-reply.gpg"
```

I can do it manually, though it's personally borderline on whether this decreases readability or not (yes my "old people" rant is that I like 80-char line lengths still).

(Running it multiple times doesn't help, I did that enough times while setting up this PR :))